d2bs/kolbot/libs/manualplay/hooks/HelpMenu.js

```diff
@@ -23,10 +23,13 @@ const HelpMenu = new function () {
     "hide": "Hide this menu",
     "make": "create config file with current characters name",
     "stash": "Stash items/gold from inventory",
+    "gamble": "Start gambling",
     "filltps": "Fill tp tome",
     "cowportal": "Make cow portal as long as bot already has leg",
     "uberportal": "Make uber portal(s) as long as bot already has key",
     "ubertrist": "Make uber tristam portal as long as bot already has organs",
+    "diablo": "Run Diablo in fast mode",
```

Comment:
I'm not sure it makes sense to have clearLevel and diablo as commands. The idea behind manualplay is just that, manually playing with some qol features. Starting up clearLevel or diablo is relinquishing control of the character.

Reply:
i like it, but agreed. removed
reverted clearlevel to original `clear` Command with  `Attack.Clear(10)`